docs/custom-commands.md

```diff
@@ -6,15 +6,15 @@ It is possible to extend fin with custom commands (per project).
 
     File `.docksal/commands/updb` (Notice **no extension**. Script name should match command name)
 
-    ```bash
+```bash
```

Comment:
Oh I see, because it falls under (1.) The problem is mkdocs does not handle the code snippet within the list. It gets rendered incorrectly.